content/docs/08-user-management/3-palette-resource-limits.md

```diff
@@ -1,63 +1,72 @@
-|Roles               |     100              |
-|CloudAccounts       |     200              |
-|Cluster profiles    |     200              |
-|Registries          |      50              |
-|Private Gateway     |      50              |
-|Api Keys            |   20 keys per user   |
-|Locations(Backup)   |      100             |
-|Certificates        |       20             |
-|Macros              |      200 (Per Project)|
-|SSH Keys            |      300              |
-|Alerts or Webhook   |   100 (Per Project)   |
-|Clusters.           |      200              |
-|Appliances (Edge hosts)|      200              |  
+# Default Palette Resource Limits
+
+
+Tenant admins can set and update resource limits for Palette. The resource limits determine the maximum number of resources that can be created in Palette. The resource limits are set at the tenant level and apply to all projects in the tenant.
+
+The following table lists the default resource limits for Palette:
+
+|Resources           |  Max Limit | Scope | 
+|--------------------|----------------------|  ---- |
+|Users               |     300              | Tenant|
+|Teams               |     100              | Tenant| 
+|Projects            |      50              | Tenant | 
+|Workspaces          |      50              | Tenant |
+|Roles               |     100              | Tenant |
+|Cloud Accounts       |     200              | Tenant |
+|Cluster Profiles    |     200              | Tenant |
+|Registries          |      50              | Tenant |
+|Private Gateway     |      50              | Tenant |
+|API Keys            |       20             |  User |
+|Backup Locations    |      100             | Tenant |
+|Certificates        |       20             | Tenant |
+|Macros              |      200              | Project|
+|SSH Keys            |      300              | Tenant |
+|Alerts or Webhook   |       100            | Project|
+|Clusters            |      10,000          | Tenant |
+|Edge Hosts          |      200            |  Tenant |
 
 # Set Resource Limit 
 
-To set resource limit:
+Use the following steps to set or update resource limits for your Palette tenant.
 
-<br />
+## Prerequisites
+
+* You must have access to the *tenant admin* role.
+
+
+## Update Limits
+
+1. Login to [Palette](https://console.spectrocloud.com) as a tenant admin.
+
+
+2. Navigate to the left **Main Menu** and select **Tenant Settings**.
 
-* Login to Palette console as `Tenant Admin`.
 
+3. Select **Resource Limits** from the **Tenant Settings Menu**.
 
-* Select `Tenant Settings` from the left ribbon menu.
 
+4. Set the values for the different Palette resources. Ensure you do not 
 
-* Select `Resource Limits` tab and set the values for different Palette resources as per tenant resource requirements.
- 
-<InfoBox>
-The resource limit can be customized according to per tenant resource requirements 
-</InfoBox>
 
-# Palette API Rate Limits
+5. Click **Save changes**.
```

Comment:
```suggestion
5. Save your changes.
```

Reply:
Offered this general sentence because otherwise, you have to say "Click the 'Save changes' button" which is more wordy.